ouroboros-consensus/src/Ouroboros/Consensus/Storage/VolatileDB/Impl.hs

```diff
@@ -231,14 +232,14 @@ openDB VolatileDbArgs { volHasFS = SomeHasFS hasFS, .. } = runWithTempRegistry $
   VolatileDB API
 ------------------------------------------------------------------------------}
 
-closeDBImpl :: IOLike m => VolatileDBEnv m blk -> m ()
+closeDBImpl :: forall m blk. (IOLike m, StandardHash blk, Typeable blk) => VolatileDBEnv m blk -> m ()
```

Comment:
```suggestion
closeDBImpl ::
     forall m blk. (IOLike m, StandardHash blk, Typeable blk)
  => VolatileDBEnv m blk
  -> m ()
```

Reply:
Also almost all functions in this module use `HasHeader blk`, so use that instead of the more verbose `StandardHash blk, Typeable blk`.